Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -858,15 +863,49 @@ void ExecuteCommand(ReplyType reply_type)
     case 0x00:  // Read Sector
     {
       const u64 dvd_offset = static_cast<u64>(s_DICMDBUF[1]) << 2;
-
       INFO_LOG_FMT(
           DVDINTERFACE,
           "Read: DVDOffset={:08x}, DMABuffer = {:08x}, SrcLength = {:08x}, DMALength = {:08x}",
           dvd_offset, s_DIMAR, s_DICMDBUF[2], s_DILENGTH);
 
       if (s_drive_state == DriveState::ReadyNoReadsMade)
         SetDriveState(DriveState::Ready);
-
+      if (DVDThread::GetDiscType() == DiscIO::Platform::Triforce)
+      {
+        if (dvd_offset & 0x80000000)
+        {
+          switch (dvd_offset)
+          {
+          // Media board status (1)
+          case 0x80000000:
+            memset(Memory::GetPointer(s_DIMAR), 0, s_DICMDBUF[2]);
```

Comment:
You can probably do this instead, both here and for the following ones:
```suggestion
            Memory::Memset(s_DIMAR, 0, s_DICMDBUF[2]);
```